.github/workflows/core_build.yml

```diff
@@ -85,8 +85,10 @@ jobs:
           restore-keys: |
             ccache:${{ matrix.os }}:${{ github.ref }}
             ccache:${{ matrix.os }}
+      - name: Install latest bash
+        run: brew install bash
       - name: Configure OS
-        run: source ./apps/ci/mac/ci-install.sh
+        run: source ./acore.sh install-deps
```

Comment:
we can now remove `/apps/ci/mac/ci-install.sh`, can't we?

Reply:
yep, sure can. added the removal of /apps/ci/mac/ci-install.sh to the pr.